PlaidLink.tsx

```diff
@@ -104,7 +104,7 @@ export default function PlaidLink({
   return (
     <WebView
       source={{
-        uri: `https://cdn.plaid.com/link/v2/stable/link.html?isWebview=true&token=${linkToken}`,
+        uri: `https://cdn.plaid.com/link/v2/stable/link.html?isWebView=true&token=${linkToken}`,
```

Comment:
Why was this changed? It looks like it should be `isWebview` according to the docs:

https://plaid.com/docs/link/webview/#create

Reply:
Good call, I thought this was part of the problem and was sure that it wasn't loading without the capital `V` but I have re-tested just now and it's working fine for me. I'll revert this part of the PR